reindexer/start_reindex.py

```diff
@@ -166,6 +170,15 @@ def start_reindex(src, dst, mode, reason):
     elif mode == "partial":
         max_records = click.prompt("How many records do you want to send?", default=10)
         parameters = partial_reindex_parameters(max_records)
+    elif mode == "specific":
+        specified_records_str = click.prompt(
+            "Which records do you want to reindex? (separate multiple IDs with spaces)",
+            type=str,
+        )
+        specified_records = specified_records_str.split()
+        if len(specified_records) is 0:
```

Comment:
Two notes:

* In Python, empty lists are false-y, so common convention is to check if a list is empty by doing:

    ```python
    if not my_list:
        print("my_list is empty!")
    ```

* `==` compares equality, `is` compares identity. Are these two objects equivalent, or are they at the same memory address?

  In general, prefer `==` unless you're sure you really want to check these are the same objects in-memory. Otherwise you can be lead astray:

    ```pycon
    >>> 1024 is 1024
    True

    >>> [] is []
    False
    >>> [] == []
    True

    >>> 10000 is 10000
    True
    >>> 10000 is int(10000.0)
    False
    >>> 10000 == int(10000.0)
    True
    ```